sql/ddl_api.sql

```diff
@@ -47,69 +47,43 @@ CREATE OR REPLACE FUNCTION  set_number_partitions(
     dimension_name          NAME = NULL
 ) RETURNS VOID AS '@MODULE_PATHNAME@', 'dimension_set_num_slices' LANGUAGE C VOLATILE;
 
--- Drop chunks that are older than a timestamp.
+-- Drop chunks that are older than a timestamp or newer than a timestamp.
 CREATE OR REPLACE FUNCTION drop_chunks(
-    older_than anyelement,
+    older_than ANYELEMENT = NULL,
     table_name  NAME = NULL,
     schema_name NAME = NULL,
-    cascade  BOOLEAN = FALSE
+    cascade  BOOLEAN = FALSE,
+    newer_than ANYELEMENT = NULL
 )
     RETURNS VOID LANGUAGE PLPGSQL VOLATILE AS
 $BODY$
 DECLARE
     older_than_internal BIGINT;
+    newer_than_internal BIGINT;
 BEGIN
-    IF older_than IS NULL THEN
-        RAISE 'the timestamp provided to drop_chunks cannot be NULL';
+    IF older_than IS NULL AND newer_than IS NULL THEN
+        RAISE 'Both older_than and newer_than timestamps provided to drop_chunks cannot be NULL';
     END IF;
-
-    PERFORM  _timescaledb_internal.drop_chunks_type_check(pg_typeof(older_than), table_name, schema_name);
-    SELECT _timescaledb_internal.time_to_internal(older_than, pg_typeof(older_than)) INTO older_than_internal;
-    PERFORM _timescaledb_internal.drop_chunks_impl(older_than_internal, table_name, schema_name, cascade);
+    PERFORM _timescaledb_internal.drop_chunks_impl(older_than, table_name, schema_name, cascade,
+    truncate_before => FALSE, newer_than_time => newer_than);
```

Comment:
What is `truncate_before` argument of drop_chunks_impl for? 
it is not ever used and there is no public facing api for it.